xds/internal/balancer/outlierdetection/balancer_test.go

```diff
@@ -1387,11 +1385,6 @@ func (s) TestConcurrentOperations(t *testing.T) {
 				t.Error("ResolverError was called after Close(), which breaks the balancer API")
 			}
 		},
-		UpdateSubConnState: func(*stub.BalancerData, balancer.SubConn, balancer.SubConnState) {
-			if closed.HasFired() {
-				t.Error("UpdateSubConnState was called after Close(), which breaks the balancer API")
```

Comment:
I'm assuming state listeners still follow this API (can't be called after close)?

Reply:
Correct!